native/cocos/scene/Skybox.cpp

```diff
@@ -42,7 +41,6 @@
 #include "scene/Model.h"
 
 namespace {
-cc::Mesh *    skyboxMesh{nullptr}; // TODO(cjh): How to release ?
 cc::Material *skyboxMaterial{nullptr};
```

Comment:
Should `skyboxMaterial` also be a member variable?

Reply:
This will also need to be refactored in the future, but it will not leak now.